xwiki-platform-core/xwiki-platform-url/xwiki-platform-url-schemes/xwiki-platform-url-scheme-standard/src/main/java/org/xwiki/url/internal/standard/AbstractWikiReferenceExtractor.java

```diff
@@ -46,6 +48,11 @@
     @Inject
     private Execution execution;
 
+    protected boolean isContextAvailable()
+    {
+        return execution.getContext() != null && execution.getContext().hasProperty(XWikiContext.EXECUTIONCONTEXT_KEY);
```

Comment:
Hmm feels like this code was not really meant to be aware of XWikiContext, I wonder if all this bulletproofing should not be moved to WikiDescriptorManager.